src/Core/Silk.NET.Core/Native/FuncPtr.cs

```diff
@@ -82,7 +82,7 @@ public FuncPtr(Delegate @delegate)
         {
             var handle = GCHandle.Alloc(@delegate);
             Value = Marshal.GetFunctionPointerForDelegate(@delegate);
-            FuncPtrInternal.Storage.Add(Value, new FuncPtrInternal {Delegate = @delegate, Handle = handle});
+            FuncPtrInternal.Storage[Value] = new FuncPtrInternal {Delegate = @delegate, Handle = handle};
```

Comment:
Note that this is not necessarily the same.

Reply:
Yeah I know, but if the handle is the same then there's not really any difference even if we override it.